android/src/main/kotlin/io/dolby/comms/sdk/flutter/mapper/AudioCaptureModeMapper.kt

```diff
@@ -43,6 +43,7 @@ class AudioCaptureModeMapper(private val audioCaptureMode: AudioCaptureMode) : M
     companion object {
         fun fromMap(map: Map<String, Any?>): AudioCaptureMode {
             val voiceFont = toVoiceFont(map["voiceFont"] as String?)
+            android.util.Log.d("[KB]", "new voice font: $voiceFont");
```

Comment:
Looks like a temporary log. Should it be removed?

Reply:
removed